doc/reference/filters.md

```diff
@@ -434,7 +434,7 @@ path:
 | template        | string | template to create request adaptor, please refer the [template](#template-of-builder-filters) for more information                                                       | No       |
 | leftDelim       | string | left action delimiter of the template, default is `{{`                                                                                                                 | No       |
 | rightDelim      | string | right action delimiter of the template, default is `}}`                                                                                                                | No       |
-NOTE: template field takes higher priority than the static field with the same name.
+**NOTE**: template field takes higher priority than the static field with the same name.
```

Comment:
Hi, maybe we should add a blank line between NOTE and table.

Reply:
yes, or it becomes part of the table.